components/HomepageSection/GetInvolved.jsx

```diff
@@ -4,72 +4,95 @@ import Link from 'next/link';
 /* eslint-disable react/prop-types */
 import SectionWrapper from 'components/Layout/SectionWrapper';
 import { Card } from 'components/ui/card';
+import SectionHeading from 'components/SectionHeading';
 
 const GET_INVOLVED_DATA = [
   {
     id: 1,
-    imageSrc: '/images/homepage/get-involved.svg',
-    title: 'Get involved',
-    description:
-      'Benefit from the Olas protocol, no \nmatter what you bring to the table.',
-    colSpan: 2,
-  },
-  {
-    id: 2,
-    imageSrc: '/images/homepage/olas-token.svg',
-    imageWidth: 277,
-    imageHeight: 172,
-    title: 'Get OLAS, use it across the network',
-    ctaText: 'Get OLAS',
-    href: '/olas-token',
-  },
-  {
-    id: 3,
     imageSrc: '/images/homepage/olas-contribute.png',
-    title: 'Grow awareness about Olas, earn points',
+    title: 'Grow awareness about Olas',
+    description:
+    'Prmote Olas on X and earn points while doing so.',
```

Comment:
there's a small typo in promote. you can easily copy the text from figma by right clicking on it 
<img width="673" alt="image" src="https://github.com/user-attachments/assets/5928ed6e-8bd7-43dd-aedf-03d87fc1dd22">

Reply:
Oh my bad 😓 Good catch